cms/djangoapps/contentstore/views/transcript_settings.py

```diff
@@ -19,6 +20,13 @@
 __all__ = ['transcript_credentials_handler']
 
 
+class TranscriptionProviderErrorType:
+    """
+    Transcription provider's error types enumeration.
+    """
+    INVALID_CREDENTIALS = 1
```

Comment:
Why not all the error types like we have in pipeline? Don't we need to handle other errors?

Reply:
We only log but don't handler other error types as they're not edx's user facing from the front-end.